File: 1008.py
n, *floors = map(int, input().split())
nowf = 0
time = 0
for i in floors:
    if nowf < i:
        time += (i - nowf) * 6 + 5
    else :
        time += (nowf - i) * 4 +5
    nowf = i
print(time, end = '')

File: 1008.cpp
#include<iostream>
using namespace std;
int main(){
	int n,a,time = 0,nowf = 0;
	cin>>n;
	for(int i = 0; i < n; i++){
		cin>>a;
		if(a > nowf){
			time = time + (a - nowf) * 6 + 5;
		}else{
			time = time + (nowf - a) * 4 + 5;
		}
		nowf = a;
	}
	cout<<time<<endl;
	return 0;
}
